<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Events</class>
 <widget class="QWidget" name="Events">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1724</width>
    <height>1110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>160</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Kirjaudu ulos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonReturnMain">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>370</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>palaa päänäyttöön</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewEvents">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>731</width>
     <height>801</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
